assets/css/map/controls/_street_view_switch.scss

```diff
@@ -0,0 +1,72 @@
+// BEM-PREFIX: LC = Leaflet Control
```

Comment:
TODO: discuss what the prefix should be (if any) or if we use another prefix instead?

Reply:
Is this `BEM-PREFIX` comment some kind of standardized thing? I can't seem to find any references to it with a quick Google search.

Reply:
Nope, just wanted to make a note of it to discuss

Reply:
I typed BEM in all uppercase and I think it just happened kinda like `// TODO:`

Reply:
I noted it on the [BEM Naming Ticket](https://app.asana.com/0/1152340551558956/1203760641228313/f) that this prefix was added in this PR so it can get addressed then.